libr/core/cmd_api.c

```diff
@@ -311,22 +312,40 @@ static int cmdstatus2int(RCmdStatus s) {
 }
 
 R_API int r_cmd_call_parsed_args(RCmd *cmd, RCmdParsedArgs *args) {
+	int res = -1;
+
+	// As old RCorePlugin do not register new commands in RCmd, we have no
+	// way of knowing if one of those is able to handle the input, so we
+	// have to pass the input to all of them before looking into the
+	// RCmdDesc tree
+	RListIter *iter;
+	RCorePlugin *cp;
+	char *exec_string = r_cmd_parsed_args_execstr (args);
+	r_list_foreach (cmd->plist, iter, cp) {
+		res = cp->call (cmd->data, exec_string);
```

Comment:
This should check if call is not NULL, that way plugins using new API can not set it and do the command registration within init and fini. Unless plugins have other init method, function where they can do the command registration.  In that case those plugins could just not define RCorePlugin.